cloudformation/ad/ad-integration.yaml

```diff
@@ -431,8 +430,13 @@ Resources:
               echo "$ADMIN_PW" | adcli create-user -x -U "${Admin}" --domain="${DirectoryDomain}" --display-name=ReadOnlyUser ReadOnlyUser
               sleep 0.5
               echo "Registering User..."
-              echo "$ADMIN_PW" | adcli create-user -x -U "${Admin}" --domain="${DirectoryDomain}" --display-name="${UserName}" "${UserName}"
-
+              NAMES="${UserNames}"
+              for name in $(echo $NAMES | sed "s/,/ /g")
```

Comment:
Let's always use self explanatory variable names. `NAMES` => `USERNAMES` and `name` => `username`